### PR TITLE
WT-3077 Fix some issues with LevelDB testing and extension pathing

### PR DESCRIPTION
**Pull request by daveh86**

### CODE REVIEW

**Comment by daveh86** [comment]
retest this please

**Comment by daveh86** [comment]
@agorrod, can you take a look at this on Thursday?

File: api/leveldb/leveldb_wt.cc [hunk 1]
```diff
@@ -248,12 +248,12 @@ leveldb::DB::Open(const Options &options, const std::string &name, leveldb::DB *
     s_conn << "exclusive,";
 #ifndef HAVE_BUILTIN_EXTENSION_SNAPPY
   if (options.compression == kSnappyCompression)
-    s_conn << "extensions=[libwiredtiger_snappy.so],";
+    s_conn << "extensions=[" EXTENSION_PATH "snappy/.libs/libwiredtiger_snappy.so],";
```
Comment (agorrod) [inline api/leveldb/leveldb_wt.cc]: I believe this change will break the code when running against an installed version of WiredTiger.

**Comment by agorrod** [comment]
Can this pull request be closed now? I don't see any changes in here.

**Comment by daveh86** [comment]
Yes. This can be

**Pull request closed without merging**